src/Uno.UI/UI/ViewManagement/InputPane/InputPane.cs

```diff
@@ -84,7 +84,11 @@ public bool TryHide()
 
 		internal void OnOccludedRectChanged()
 		{
+#if __ANDROID__
```

Comment:
it seems odd to me that we return KeyboardRect. For Android are we using OccludedRect at all then?

Reply:
Was thinking the same, NavigationBarRect is not needed at all for android for the inputpane ?

Reply:
Was thinking the same, NavigationBarRect is not needed at all for android for the inputpane ?

Reply:
I think you right. I will add some UI tests but it's probably the wrong place to fix it

Reply:
Thanks in advance for the UI tests ;) !

Reply:
Both the navigation bar and keyboard may be blocking the bottom area. Hence why they are both being propagated from here. OccludedRect should be used to calculate bottom offset. And, keyboard and navigation rects are used to check their visibility.

Reply:
Indeed. As I said above, this is the wrong place to fix this bug.
I've found that NavigationRect is always equals to 48, regardless of navigation bar visibility (I've found that thanks to the UI Tests). I didn't have time to fix it this week but I hope I'll be able to do it the next one.